### DISCUSSION TITLE
Developer Roadmaps: 安排你的學習之旅

### DISCUSSION THREAD

**Discussion by OG-Matcha** [1101-GS4719-C/Python-Community-GS4719-C#1912] [General]
我之前在尋找 Learning Path 的時候發現這個[網站](https://roadmap.sh/)

![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/85878531/e8df32f2-2a96-47c6-be8a-23288f7ceec0)

他裡面從兩種面向，分別是「角色」和「技能」來對學習路徑做分別，內容有持續更新，裡面也囊括了許多技術的學習路徑。

我覺得對於想了解特定領域或技能的人來說可以知道自己還需要學習那些技術非常的方便，他也有提供學習紀錄還有逐漸擴充的 Practices 和 Questions 測驗，提供給各位參考。

**Comment by benbebobo149** [comment]
我認為，在當今資訊發達的時代，想要學習新技能或領域的人有許多資源可供選擇。然而，資源眾多也帶來了新手容易迷失的問題。對我而言，擁有一個清晰的學習路徑至關重要，這可以幫助我避免在學習過程中迷失方向，並保持學習的動力。
對我而言，這個網站提供了一個很好的學習指南，讓我能夠更清楚地了解自己在學習特定領域或技能時需要掌握的知識和技術。感謝同學的分享，讓我發現了這個寶貴的資源。

**Comment by lyykey** [comment]
對於我個人而言，Developer Roadmaps 提供了一個非常寶貴的參考框架，幫助我在技術學習和職業發展方面有所指引。它不僅僅是一個學習資源，更是一個引導我們在不斷變化的科技領域中找到方向的工具。我相信，這個網站將會為許多開發者提供寶貴的幫助，讓他們能夠更有效地前進並實現自己的目標。

**Comment by 155Julia** [comment]
感謝分享！這學習路徑設計很貼心，能讓我清楚了解技能水平並制定計劃。已經開始探索，覺得內容豐富且更新及時。期待透過此資源不斷擴充知識和技能，特別是學習記錄和測驗功能。謝謝你的分享！

**Comment by YeeDaniel** [comment]
謝謝這位同學的分享，了解到在學習不同的技能有一個Roadmap可以跟隨，並且如同樓上同學所說裡面的內容很豐富且即時更新。同時，這也會對許多的開發者帶來許多的便利性，他也會成為開發者學習路上的一盞明燈。

**Comment by yyujimmy** [comment]
以下是我瀏覽網頁後的想法：
這個資源不僅提供了詳盡的內容，還將引導到一個全新的學習體驗。我們知道在這個數字時代，資訊變得無所不在，但這個資源與眾不同。它以一種清晰、易於理解的方式呈現了複雜的概念，無論你的經驗水平如何，你都能從中受益良多。這個資源的品質和內容表明了他們對於教育和知識分享的承諾，我深信這將會激勵更多人追求知識和共同成長。

**Comment by wanderer657** [comment]
感謝分享，我發現學新東西時，選對學習資源超重要。其他的都亂七八糟，但這網站給了一條清晰的路線。感謝兄弟分享，讓我找到這超棒的東西。用它，我學得可比以前快多了！我覺得這網站真是個學習的神器，比起其他工具或方法，它真的效率高太多了。

**Comment by Eileen050824** [comment]
很棒的學習路徑分享！之前想開始自學一些基礎程式語言的時候，大多是看書自學python，後面又有親友推薦一些線上學習資源，又加上一些課程作業需求碰了一下C語言和C++，到後來有一點點混亂，也對我學程式語言的目的些許迷失。但這個網頁會讓你更清楚知道你的需求，順便幫助你取捨並更聚焦在你的學習核心，對我這個資訊小白提供蠻大的幫助！
